packages/sources/svelte/package.json

```diff
@@ -109,7 +110,8 @@
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
   ],
-  "main": "src/index.js",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
   "svelte": "src/index.js",
```

Comment:
No change here ?

Reply:
Nothing change here. In fact, instead of using the component not compiled, we used the compiled one.
Before, we have to use the src index because the dist doesn't expose an index.js who export all components as js.

Reply:
I mean, why not `"svelte": "dist/index.js",` like the others entries.
Like this, we always use by default the `/dist` folder.

Reply:
Yes, I will check that, but if I do that, I need to reimport the `@vtmn` as dependances ^^